=== RUN   TestA01_SendOnClosed
--- FAIL: TestA01_SendOnClosed (0.00s)
panic: send on closed channel [recovered]
	panic: send on closed channel

goroutine 18 [running]:
testing.tRunner.func1.2({0x551040, 0x5b1400})
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1734 +0x21c
testing.tRunner.func1()
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1737 +0x35e
panic({0x551040?, 0x5b1400?})
	/home/ilian/Tools/go-runtime/src/runtime/panic.go:792 +0x132
scenarios.TestA01_SendOnClosed(0xc000102700?)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go:18 +0x37
testing.tRunner(0xc000102700, 0x585bb8)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
created by testing.(*T).Run in goroutine 1
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1851 +0x413
FAIL	scenarios	0.004s
FAIL
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go
TestName:  TestA01_SendOnClosed
Import added at line: 3
Header added at line: 15
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA01_SendOnClosed

Exit Replay with code  30 Send on close
Exit Message: panic: send on closed channel

FAIL	scenarios	0.103s
FAIL
